docs/guides/primitives-rest-api.mdx

```diff
@@ -479,6 +479,143 @@ else:
   </TabItem>
 </Tabs>
 
+<span id="start-sampler-parms"></span>
+## Sampler primitive with REST API and parameterized circuits
+
+### 1. Initialize the account
+
+Because Qiskit Runtime is a managed service, you first need to initialize your account. You can then select the device you want to use to run your calculations on.
+
+Find details on how to initialize your account, view available backends, and invalidate tokens in this [topic](./setup-channel#set-up-to-use-ibm-quantum-platform-with-rest-api).
+
+### 2. Define parameters
+
+```python
+import requests
+import qiskit_ibm_runtime
+from qiskit_ibm_runtime import QiskitRuntimeService
+from qiskit.transpiler.preset_passmanagers import generate_preset_pass_manager
+from qiskit.qasm3 import dumps
+from qiskit import QuantumCircuit
+from qiskit.circuit import Parameter
+from qiskit import transpile
+
+service = QiskitRuntimeService(channel='ibm_quantum')
+backend = service.backend("<your selected backend>")
+
+pm = generate_preset_pass_manager(backend=backend, optimization_level=1)
+
+theta = Parameter('theta')
+phi = Parameter('phi')
+parameter_values = {'theta': 1.57, 'phi': 3.14}   # In case we want to pass a dictionary
+```
+
+
+### 3. Create a quantum circuit and add parameterized gates
+
+```python
+qc = QuantumCircuit(2)
+
+# Add parameterized gates
+qc.rx(theta, 0)
+qc.ry(phi, 1)
+qc.cx(0, 1)
+qc.measure_all()
+
+# Draw the original circuit
+qc.draw('mpl')
+
+# Get an ISA circuit
+isa_circuit = pm.run(qc)
+```
+
+### 4. Generate QASM 3 code
+
+```python
+qasm_str = dumps(isa_circuit)
+print("Generated QASM 3 code:")
+print(qasm_str)
+```
+
+### 5. Run the quantum circuit using Sampler V2 API
+
+
+<Admonition type="note">
+    The jobs below use Qiskit Runtime V2 [primitives](/guides/primitives). Both [`SamplerV2`](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.SamplerV2) and [`EstimatorV2`](/api/qiskit-ibm-runtime/qiskit_ibm_runtime.EstimatorV2) take one or more [primitive unified blocs (PUBs)](/guides/primitive-input-output#pub-overview) as the input. Each PUB is a tuple that contains one circuit and the data broadcasted to that circuit, which can be multiple observables and parameters. Each PUB returns a result.
+</Admonition>
+
+
+```python
+import requests
+
+url = 'https://api.quantum-computing.ibm.com/runtime/jobs'
+headers = {
+    'Content-Type': 'application/json',
+    '"Authorization": f"Bearer xxxxxxx",
+    'x-qx-client-application': 'qiskit-version-2/0.39.2/'+'your_application' # specifying the application you might be running from. For an actual Integration project, this option is invaluable to know where jobs are coming from. At this time the "qiskit-version-2/0.39.2/" string is a necessary prefix.
```

Comment:
Can you please double check with the SMEs `qiskit-version-2/0.39.2` is still accurate - I'm surprised it's set to 0.39.2?

```suggestion
    # You should replace <your_application> with the name of your app. This is useful to know
    # where jobs are coming from. Note that the prefix "qiskit-version-2/0.39.2/" is necessary.
    'x-qx-client-application': 'qiskit-version-2/0.39.2/<your_application>',
```

Reply:
I still suggest my change to use an f string and put the comment above the line, split over two lines

Reply:
Yep - I wasn't ignoring that comment; I was just leaving it open because I'm still looking for a good answer from an SME.